src/components/filopplaster/filopplasterModal/FilopplasterModal.tsx

```diff
@@ -108,8 +146,12 @@ const FilopplasterModal: React.FC = () => {
         })
         .then(() => {
           settLaster(false);
+          settDato(null);
+          settBeløp(null);
+          settTransportmiddelKvittering(undefined);
+          settUopplastetFil(null);
+          settHarAlleredeBlittValidert(false);
           lukkModal();
-          settTransportmiddel(undefined);
         })
         .catch((error) => {
           logger.error('Feil under opplasting av kvittering', error);
```

Comment:
Bør vi ha en feilmelding til brukeren på dette òg? Kan kanskje gjøres til eget kort i så fall

Reply:
Usikker. Det her er jo etter at alt som bruker har gjort har blitt validert, så tanken med å logge den feilen var for mest for utviklere.